test/integration/oauth_serviceaccount_client_test.go

```diff
@@ -160,26 +160,8 @@ func TestOAuthServiceAccountClient(t *testing.T) {
 			t.Fatalf("Unexpected error: %v", err)
 		} else if !reflect.DeepEqual(clientAuth.Scopes, []string{"user:full"}) {
 			t.Fatalf("Unexpected scopes: %v", clientAuth.Scopes)
-		} else {
-			// update the authorization to not contain any approved scopes
-			clientAuth.Scopes = nil
```

Comment:
Set this to something minimal like `user:info` and leave the rest of the test intact.